_content/av/early-buddhist-tok-course_sujato.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Early Buddhist Theory of Knowledge"
+authors:
+  - sujato
+subcat: course
+external_url: "https://www.youtube.com/@BuddhistLibrarySydney/search?query=theory%20of%20knowledge"
+drive_links:
+  - "https://drive.google.com/drive/folders/1IQmEOeL-y5ZN1OCHMnyF8b_76mBfiqLT"
+formats: ["mp3s"]
+status: featured
+course: epistemology
+tags:
+  - empiricism
+  - phenomenology
+  - logic
+  - interfaith
+year: 2021
+publisher: "Buddhist Library"
+minutes: 583
+---
+
+This is a five-part series on Jayatilake's "Early Buddhist Theory of Knowledge"[(/content/monographs/early-buddhist-theory-of-knowledge_jayatilleke)] given by Bhante Sujato. The main topics are: 1. the historical background of non-Buddhist thought at the time of the Buddha; 2. the use and limits of logic; 3. the roles of authority and reason; and 4. the limits of knowledge.
```

Comment:
To make numbered lists in markdown, put each numbered item on a new line. Not a bad description! More info on the course can be found on the course info pages [on the SBL](https://buddhistlibrary.org.au/events/early-buddhist-theory-of-knowledge/) and [SC:D&D](https://discourse.suttacentral.net/t/early-buddhist-theory-of-knowledge-course-outline-for-the-buddhist-library/21213?u=khemarato.bhikkhu) websites.